packages/tx-construction/src/tx-builder/OutputBuilder.ts

```diff
@@ -116,6 +120,21 @@ export class TxOutputBuilder implements OutputBuilder {
     return this;
   }
 
+  // make it take an output that has type AdaHandle
+  handle(_output: Handle): OutputBuilder {
+    // output/handle: $AdaHandle
```

Comment:
I imagine none of this set-up should be merged. So I can delete it, but I wanted to initially leave it, so we see how if I do the handle here, as opposed to txBuilder, I can continue chaining actions. But if I move it to txBuilder, then `.....handle('$AdaHandle').coin().` wouldn't work. 
Also,

Reply:
See